src/geo/map.js

```diff
@@ -90,6 +90,13 @@ var Map = Model.extend({
 
   // PUBLIC API METHODS
 
+  moveCartoDBLayer: function (from, to) {
+    var layerMoved = this.layers.moveCartoDBLayer(from, to);
+    if (layerMoved) {
+      this.trigger('cartodbLayerMoved', {}, this);
```

Comment:
Because with listenTo, the bind is removed when Class is eliminated (http://stackoverflow.com/questions/16823746/backbone-js-listento-vs-on).